.github/workflows/build-windows.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       - name: compile
         run: |
-          $(env:GITHUB_WORKSPACE)\DevelopmentEnvironment\gcc\Msys_x2\1.0\msys.bat'
+          Invoke-Command -FilePath '$(env:GITHUB_WORKSPACE)\DevelopmentEnvironment\gcc\Msys_x2\1.0\msys.bat'
```

Comment:
If you want to run msys.bat, you must use this syntax:
`& "$env:GITHUB_WORKSPACE\DevelopmentEnvironment\gcc\Msys_x2\1.0\msys.bat"`